### ISSUE TITLE
How to compare loops of Hi-C from different conditions?

### ISSUE THREAD

**Issue by Linhua-Sun** [ay-lab/mustache#21]
Hi: 
    Thanks for your great works! I am wondering how to compare the loops quantitatively？For example, if we got two Hi-C samples under different conditions. 
    Option 1: we can call loops for A and B Hi-C separately. Then we can merged the LoopA and LoopB (sometimes, loops results differ too much which is inconsistent with hic results in juicebox ). Nest step, **how should we calculate the relative contact intensity for each loops in LoopA+LoopB**. Just DEG analysis in RNA-Seq, can we identify those gain/loss loops in different conditions?
    Option 2: we can merge the two hic files and call loops in an artificial merged hic samples. Then key question is also how to calculate the contact intensity for each loops and found those significantly differed?
    **In summary, how to call loops for two or more Hi-Cs and how to compare them?**
    Thank you for your time.
Best wishes!
Linhua

**Comment by ay-lab** [comment]
Sorry, I don't know of any standard way of quantitatively comparing loops of different conditions, cell types etc. You can simply say these loops are present in this condition and not in the other but for comparing their intensities maybe you can compare O/E average in a small neighborhood around the loops and compare. For directly comparing the whole hi-c contact maps though you can use methods like hicdc+ or selfish.

**Comment by Linhua-Sun** [comment]
Thank you so much!
